components/nifosg/nifloader.cpp

```diff
@@ -2083,8 +2079,10 @@ namespace NifOsg
 
             mat = shareAttribute(mat);
 
+            osg::StateSet* stateset = node->getStateSet();
```

Comment:
The game crashes while loading because this is a nullptr.

Reply:
Crashes for me as well. I suggest to revert ASAP, before it get into nightly builds.

Reply:
Reverted.